zubhub_frontend/zubhub/src/views/activity_details/ActivityDetailsV2.jsx

```diff
@@ -260,22 +260,18 @@ export default function ActivityDetailsV2(props) {
         </Typography>
 
         <Grid container spacing={4} justifyContent="center">
-          {moreActivities.map((activity, index) => (
-            <Grid
-              key={index}
-              item
-              {...(auth.token ? authenticatedUserActivitiesGrid : unauthenticatedUserActivitiesGrid)}
-              align="center"
-            >
-              <Activity
-                activity={activity}
-                key={activity.id}
-                t={t}
-                // updateProjects={res => handleSetState(updateProjects(res, state, props, toast))}
-                {...props}
-              />
-            </Grid>
-          ))}
+          {moreActivities
+            .filter(({ id }) => id !== props.match.params.id)
```

Comment:
simple change such that the currently browsed activity doesn't shore in the `moreActivities` section